ydb/library/yql/dq/actors/spilling/spilling_file.cpp

```diff
@@ -712,6 +734,55 @@ class TDqLocalFileSpillingService : public TActorBootstrapped<TDqLocalFileSpilli
         Send(ev->Sender, new NMon::TEvHttpInfoRes(s.Str()));
     }
 
+    void HandleWork(TEvPrivate::TEvRemoveOldTmp::TPtr& ev) {
+        const auto& msg = *ev->Get();
+        const auto& root = msg.TmpRoot;
+        const auto nodeIdString = ToString(msg.NodeId);
+        const auto& sessionId = msg.SpillingSessionId;
+
+        LOG_I("[RemoveOldTmp] removing at root: " << root);
+
+        static const auto isDirOldTmp = [&nodeIdString, &sessionId](TString dirName) -> bool {
+            // dirName: node_<nodeId>_<sessionId>
+            static constexpr size_t NodeIdBegin = 5;
+            if (dirName.Size() < NodeIdBegin || dirName.substr(0, NodeIdBegin) != "node_") {
+                return false;
+            }  
+            const auto nodeIdEnd = dirName.find('_', NodeIdBegin);
```

Comment:
nit: this code may be simplified. You can split the string by "_" and check if vector size equals to 3. Then compare the elements of vector with the expected ids